.github/workflows/release.yml

```diff
@@ -0,0 +1,47 @@
+name: Release
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    name: Release
+    env:
+      TAR_FILE_NAME: stargz-snapshotter-linux-x86_64.tar.gz
```

Comment:
s/x86_64/amd64/ to match containerd releases: https://github.com/containerd/containerd/releases/tag/v1.4.1

Reply:
Also consider adding version number to file name